src/audio/heap.c

```diff
@@ -976,9 +976,6 @@ s32 audio_shut_down_and_reset_step(void) {
 void wait_for_audio_frames(s32 frames) {
     // VC emulator stubs this function because busy loops are not supported
     // Technically we can put infinite loop that _looks_ like -O0 for emu but this is cleaner
-    //if (gIsVC) {
-        return;
```

Comment:
why is this removed, is this still confirmed to work with VC

Reply:
This wouldn't cause issues with VC, it would cause issues (theoretically) for console. But I believe it just causes lag.

Reply:
This function goes unused anyway. The reason we haven't deleted it to begin with is because supposedly VC might be able to still find it? Idk how that part works.

Reply:
VC can only match strictly binary functions so it just does nothing atm and can be removed.

Reply:
whats the reason to remove just this line and nothing else then if its unused

Reply:
To make it perfectly match vanilla, no other reason. But given what aglab said, we should probably just take it out.

Reply:
so what happened with this